dev/sbt-plugin/src/sbt-test/sbt-plugin/http-backend-switch/apis/src/main/java/com/example/hello/api/AkkaHttpService.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *
+ */
+package com.example.hello.api;
+
+import akka.NotUsed;
+import com.lightbend.lagom.javadsl.api.Descriptor;
+import com.lightbend.lagom.javadsl.api.Service;
+import com.lightbend.lagom.javadsl.api.ServiceCall;
+
+import static com.lightbend.lagom.javadsl.api.Service.named;
+import static com.lightbend.lagom.javadsl.api.Service.pathCall;
+
+/**
+ * The hello service interface.
+ * <p>
+ * This describes everything that Lagom needs to know about how to serve and
+ * consume the AkkaHttpService.
```

Comment:
I think it would be better to delete the boilerplate comments... they don't add information so it's just noise.